pynio/block.py

```diff
@@ -118,3 +118,16 @@ def copy(self, name, instance=None):
         out._name = name
         out.config['name'] = name
         return out
+
+    def in_use(self, instance=None):
+        '''returns a dictionary of services that use this block
+
+        If instance is None (default) uses it's own instance
+        '''
+        if instance is None:
+            instance = self._instance
+
+        name = self.name
+        return {sname: service for (sname, service) in instance.services.items()
```

Comment:
I know I said to return a dictionary, but now I'm thinking it might make more sense for this to just be a list. What do you think? Any reason for the dictionary? 

instance.blocks and instance.services return dictionaries and I wanted to be consistent with that. But now I'm thinking lists might be the way to go.